graylog2-server/src/main/java/org/graylog2/indexer/IndexSetValidator.java

```diff
@@ -134,10 +134,14 @@ public Violation validateRefreshInterval(Duration readableDuration) {
 
     @Nullable
     private Violation validatePrefix(IndexSetConfig newConfig) {
+        if (newConfig.indexPrefix().contains(MongoIndexSet.KEYWORD_WARM)) {
```

Comment:
```suggestion
        if (newConfig.indexPrefix().equals(MongoIndexSet.KEYWORD_WARM)) {
```
I think this is too strict. Allowing `warm` in a word won't hurt us